cibuildwheel/__main__.py

```diff
@@ -361,35 +363,12 @@ def main() -> None:
             assert_never(platform)
 
 
-def detect_obsolete_options() -> None:
-    # Check the old 'MANYLINUX1_*_IMAGE' options
-    for (deprecated, alternative) in [
-        ("CIBW_MANYLINUX1_X86_64_IMAGE", "CIBW_MANYLINUX_X86_64_IMAGE"),
-        ("CIBW_MANYLINUX1_I686_IMAGE", "CIBW_MANYLINUX_I686_IMAGE"),
-    ]:
-        if deprecated in os.environ:
-            print(
-                f"'{deprecated}' has been deprecated, and will be removed in a future release. Use the option '{alternative}' instead."
-            )
-            if alternative not in os.environ:
-                print(f"Using value of option '{deprecated}' as replacement for '{alternative}'")
-                os.environ[alternative] = os.environ[deprecated]
-            else:
-                print(f"Option '{alternative}' is not empty. Please unset '{deprecated}'")
-                sys.exit(2)
-
-    # Check for deprecated identifiers in 'CIBW_BUILD' and 'CIBW_SKIP' options
-    for option in ["CIBW_BUILD", "CIBW_SKIP"]:
-        for deprecated, alternative in [
-            ("manylinux1", "manylinux"),
-            ("macosx_10_6_intel", "macosx_x86_64"),
-            ("macosx_10_9_x86_64", "macosx_x86_64"),
-        ]:
-            if option in os.environ and deprecated in os.environ[option]:
-                print(
-                    f"Build identifiers with '{deprecated}' have been deprecated. Replacing all occurences of '{deprecated}' with '{alternative}' in the option '{option}'"
-                )
-                os.environ[option] = os.environ[option].replace(deprecated, alternative)
+def deprecated_selectors(name: str, selector: str, *, error: bool = False) -> None:
+    if "p27" in selector or "p35" in selector:
```

Comment:
```suggestion
    if "p2" in selector or "p35" in selector:
```
Should it be `p2` instead of `p27` ?

Reply:
Probably could be, just in case. It just has to only match the correct place, but I'm pretty sure no other parts of our selectors have a p2?